git-cliff-core/src/changelog.rs

```diff
@@ -186,6 +145,58 @@ impl<'a> Changelog<'a> {
 		Ok(())
 	}
 
+	fn process_commit_list(
```

Comment:
This is new, because commit preprocessing now needs to be done for release commits AND submodule commits.